doc/util.md

```diff
@@ -1,37 +1,39 @@
 # Util
 
-Some stuff just doesn't fit anywhere else.
+On this page you can find documentation for some deviceities classes and methods.
```

Comment:
`deviceities` is not a word in my book ;-). I suspect you meant something about devices?

Reply:
Lol, I think I mixed utilities and device, thanks for spotting that

Reply:
Oh, I see what happened, I was quite naive on trusting that a replace all would fix everything quicker! hahaha